causalml/inference/tree/uplift.pyx

```diff
@@ -1256,7 +1260,8 @@ class UpliftRandomForestClassifier:
                  n_reg=10,
                  evaluationFunction='KL',
                  normalization=True,
-                 n_jobs=-1):
+                 n_jobs=-1,
+                 joblib_prefer: str = "threads"):
```

Comment:
nit: We're going to use [`black`](https://black.readthedocs.io/en/stable/) as a formatter as discussed in #474. You can use it integrated in your IDE or run its command line tool to ensure that the code style follows [PEP-8](https://www.python.org/dev/peps/pep-0008/#other-recommendations). e.g. no white space around `=` when used in input arguments.

Since we already have a separate PR, you don't need to update the style here.